src/core/num_co.cc

```diff
@@ -399,7 +399,7 @@ Real_mv clasp_floor2(Real_sp x, Real_sp y) {
     }
     break;
   case number_SingleFloat: { /* SF / ANY */
-    float n = clasp_to_double(y);
+    float n = clasp_to_double(gc::As<Number_sp>(y));
```

Comment:
Here y is a T_sp - so yes - use gc::As<Number_sp>(y)

Reply:
ok